### ISSUE TITLE
error: missing ), unterminated subpattern at position x

### ISSUE THREAD

**Issue by mikelam14** [Pandora-Intelligence/concise-concepts#13]
When I run the example code snippet with my custom data, during the step of adding step to spacy pipeline i ran into the following error:
`error: missing ), unterminated subpattern at`

To reproduce:
```
few_shot = {
    "soccer": ["ronaldo", "messi"]
}
nlp = spacy.load("en_core_web_lg", disable=["ner"])
nlp.add_pipe("concise_concepts", config={"data": few_shot})
```

**Comment by davidberenstein1957** [comment]
Thanks, I resolved this issue in `0.6.1`.

**Comment by mikelam14** [comment]
Thank you, can confirm it works now.
